io-engine/src/bdev/nvmx/qpair.rs

```diff
@@ -200,7 +200,6 @@ impl QPair {
         // multiple allocations of the same handle for the same thread, to make
         // sure we don't reconnect every time.
         if self.state() == QPairState::Connected {
-            trace!(?self, "I/O qpair already connected");
```

Comment:
Why is this removed, seems useful to have?

Reply:
This looks like a mistake after I had modified these traces during testing. Will put back.